include/triggeralgs/TriggerDecision.hpp

```diff
@@ -17,9 +17,9 @@
 namespace triggeralgs {
 struct TriggerDecision
 {
-  int64_t time_start = { 0 };
-  int64_t time_end = { 0 };
-  int64_t time_trigger = { 0 };
+  timestamp_t time_start = { 0 };
+  timestamp_t time_end = { 0 };
+  timestamp_t time_trigger = { 0 };
   uint32_t trigger_number = { 0 }; // NOLINT(build/unsigned)
   uint32_t run_number = { 0 };     // NOLINT(build/unsigned)
   uint32_t subrun_number = { 0 };  // NOLINT(build/unsigned)
```

Comment:
Perhaps `number_t` `count_t` or `id_t` here? Also, is `uint32_t` sufficient for trigger number?

Reply:
I left `TriggerDecision` out because I'm not sure what is going to happen to it. The code in the `trigger` package uses the `TriggerDecision` class defined in `dfmessages` (since the TD is passed to the dataflow subsystem):

https://github.com/DUNE-DAQ/dfmessages/blob/develop/include/dfmessages/TriggerDecision.hpp

But I added a `using trigger_number_t = uint64_t`

Reply:
`Set.hpp` in `trigger` depends on `Type.hpp` from `dataformats` for its `dataformats::timestamp_t`.  I think it should change to using the equivalent type from `triggeralgs' Type.hpp`.

Reply:
Done in https://github.com/DUNE-DAQ/trigger/commit/6a2b33261eb3042ae758fefdc52519fcf9a6e76d